components/tooltip.scss

```diff
@@ -1,98 +1,121 @@
-$tooltip-tip-size: 10px;
-$tooltip-padding: 0.5rem 1rem;
-$tooltip-width: 150px;
-$tooltip-menu-side-padding: 0.75rem;
-$tooltip-menu-padding: 0.5rem 1rem;
-$tooltip-tip-right-offset: 1rem;
-$tooltip-tip-left-offset: 1rem;
-$tooltip-border-radius: $border-radius;
-
 .hdd-tooltip {
-  // Variable containing '&' to jump to current parent inside a child class.
   $this: &;
-
   position: relative;
+  display: inline-block;
 
-  &:hover:not(.no-hover) {
+  &:hover {
+    #{$this}_content {
+      @extend .hdd-tooltip_content, .is-active;
+    }
+  }
+  &:focus {
     #{$this}_content {
       @extend .hdd-tooltip_content, .is-active;
     }
   }
 
   &_content {
-    //.is-p is font-properties, defined in typography.
-    @extend .is-p;
-
-    $content-color: var(--hdd-color-grey-dark);
-    $content-text-color: var(--hdd-color-white);
-
-    display: table;
+    @extend .is-p-small;
     position: absolute;
-    z-index: -1;
-
-    top: calc(100% + #{$tooltip-tip-size});
-    right: 0;
-    left: 50%;
-
-    box-sizing: border-box;
-    min-width: $tooltip-width;
-
-    margin: auto;
-    padding: $tooltip-padding;
-
-    transform: translateX(-50%);
-    transition: opacity $medium-transition $transition-cubic, z-index $medium-transition step-end;
-    border-radius: $tooltip-border-radius;
-
     opacity: 0;
-    background: $content-color;
-    color: $content-text-color;
-    text-align: left;
-
-    &::before {
-      content: '';
-
-      display: block;
-      position: absolute;
-
-      top: -$tooltip-tip-size;
-      right: 0;
-      left: 0;
-
-      width: 0;
-      height: 0;
-
-      margin: auto;
-
-      border-right: $tooltip-tip-size solid transparent;
-      border-bottom: $tooltip-tip-size solid $content-color;
-      border-left: $tooltip-tip-size solid transparent;
+    z-index: -1;
+    min-width: 90px;
+    max-width: 70ch;
```

Comment:
hvorfor ch?

Reply:
Har følgt den 70-80 ch bredde for god lesbarhet. Men vi kan også endre denne om det er bedre med lengre/kortere i henhold til at det er bare en liten tooltip?